addons/crm_livechat/models/mail_channel.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import re
+from odoo import models, _
+
+
+class MailChannel(models.Model):
+    _inherit = 'mail.channel'
+
+    def _define_command_lead(self):
+        return {'help': _('Create a new lead')}
+
+    def _execute_command_lead(self, **kwargs):
+        partner = self.env.user.partner_id
+        msg = _('Something is missing or wrong in command')
+        if kwargs.get('body'):
+            key = kwargs['body'].split()
+            channel_partners = self.env['mail.channel.partner'].search([('partner_id', '!=', partner.id), ('channel_id', '=', self.id)], limit=1)
+            if key[0].lower() == '/lead':
+                if len(key) == 1:
+                    if self.channel_type == 'channel':
```

Comment:
@tivisse Remove this all useless conditions, we just check conditions for inform user where he/she is? I don't think we really need it.